app/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
     minSdkVersion rootProject.ext.minSdkVersion
     targetSdkVersion rootProject.ext.targetSdkVersion
     versionCode 1
```

Comment:
Shouldn't we change this too?

Reply:
it takes only integer values and is mostly for play-store etc to track version. I don't see value in it atm.